documentation/api/query/v4/pql.markdown

```diff
@@ -39,8 +39,8 @@ A PQL query has the following structure:
 
 Which is broken up into the following parts:
 
-* `entity`: Rquired. The entity context that this query executes on.
-* `projection`: Optional. Modifies the output of how the `entity` fields are returned.
+* `entity`: Required. The entity context that this query executes on.
+* `projection`: Optional. Restricts the output to a selection of fields.
```

Comment:
The new wording doesn't quite describe that it covers functions also, and later with DAP (document aware projection) we'll need to extend it again. I know you're better at describing things than I am, perhaps a different wording here? Maybe just saying 'a selection of fields or functions' or something like that?

Tangent/Unrelated: Also fwiw, lots of people hate the word 'projection' even if it is correct. I've had this feedback directly from a large number of people. @nfagerlund, not sure if you have any ideas around this also.